docs/2_1_common_api.adoc

```diff
@@ -580,6 +580,28 @@ For Co-Simulation FMUs, additional functions are defined in <<transfer-of-input-
 
 The value of a variable may only be accessed with the respective `fmi3Get/Set{VariableType}` for its type.
 
+==== Advancing Time [[advancing-time]]
+
+This section highlights the differences of the concept of time (in general the independent variable) for the three different FMI types, ME, CS and SE.
+
+In Model Exchange, time is under the sole control of the importer and its integration algorithm.
+The model itself receives the current time to be used in its computation with <<fmi3SetTime>>.
+In fact, time is not necessarily advancing linearly as solvers might need to find zero-crossing points or the importer uses the Model Exchange FMU to find points of optimal control.
+
+In Co-Simulation, time advances in variable steps negotiated between the co-simulation algorithm of the importer and the FMU.
+The importer calls <<fmi3DoStep>> with the <<currentCommunicationPoint>> and a target <<communicationStepSize>> (required to be larger than 0.0).
+During this <<fmi3DoStep>>, both importer and FMU might encounter events that will reduce the <<communicationStepSize>> (potentially even down to 0.0).
+With the mechanisms of the callback <<fmi3CallbackIntermediateUpdate>> and/or <<earlyReturn>>, both importer and FMU can determine if such a step-size reduction is required.
+The output argument <<lastSuccessfulTime>> of <<fmi3DoStep>> allows the FMU to signal the master its current internal time.
+
+In Scheduled Execution, time has a more discrete form.
+The scheduler of the importer activates specific tasks according to the time of the importer.
+However, the time itself is not communicated to the FMU nor its partitions.
```

Comment:
"However, the time itself is not communicated to the FMU nor its partitions." ?? We communicate the activation time to the model partition.

Reply:
thanks (I completely missed this feature!!), fixed